### ISSUE TITLE
Just making sure I understand...

### ISSUE THREAD

**Issue by eloch216** [ebimodeling/biocro-dev-documentation#1]
Presumably many of the links in this repository's `README.md` will be removed once the relevant PRs have been approved and merged into the main branch of `ebimodeling/biocro-dev`? And later, this page may have links to documentation for older versions of BioCro, in case someone is still using them? Is this all a manual process?

**Comment by gsrohde** [comment]
Yes, I only added these links to make it easier (1) for reviewers to see the changed documentation, and (2) to keep track of documentation-related PRs.

(1) is probably really not necessary, since I can just give a direct link in the PR.  And there are other, possibly better, ways to accomplish (2).  But I haven't really been worrying too much about how this README page looks, since this is a "dev" version after all.

For the public version of BioCro, I imagine linking only to formal releases, with a special link to the latest release.  I take the [PEcAn project's documentation](https://pecanproject.github.io/documentation.html) as a model.

Yes, it's all a manual process.  If, when we move this all to the public version, revising links on the README page only has to be done when we do a new release, it doesn't seem too odious a maintenance task.  This, by the way, is the sort of information I imagine might go into a BioCro maintenance manual—which I imagined could be something separate from the developer's manual—should we wish to bother to make one.  (See [this comment](https://github.com/ebimodeling/biocro-dev/pull/471#issuecomment-1337906835).)
